steps/delius/utils/date-time.ts

```diff
@@ -2,6 +2,7 @@ import { addMonths, addYears, addDays, subMonths, subDays } from 'date-fns'
 export const options = { day: 'numeric', month: 'long', year: 'numeric' } as Intl.DateTimeFormatOptions
 export const recallDateFormatter = date => date.toLocaleDateString('en-GB', options)
 export const DeliusDateFormatter = (date: Date) => date.toLocaleDateString('en-GB')
+export const OasysDateFormatter = (date: Date) => date.toLocaleString('en-GB').split(',')[0].replace(/\//g, '')
```

Comment:
Maybe be simpler to use the date-fns format function here?
```suggestion
export const DeliusDateFormatter = (date: Date) => format(date, 'dd/MM/yyyy')
export const OasysDateFormatter = (date: Date) => format(date, 'ddMMyyyy')
```